src/main/scala/mesosphere/marathon/MarathonSchedulerDriver.scala

```diff
@@ -69,6 +69,14 @@ object MarathonSchedulerDriver {
     if (config.isFeatureSet(Features.TASK_KILLING))
       frameworkInfoBuilder.addCapabilities(Capability.newBuilder().setType(Capability.Type.TASK_KILLING_STATE))
 
+    // GPU Resources feature gives Marathon capability to get GPU related resources offers from Mesos. In current,
+    // key "gpus" in Marathon's API can only work with GPU Resources feature enabled. For details of GPU Resource
+    // feature, see MESOS-5634.
```

Comment:
I would write this a bit different, because no offers would get send by agents with GPUs: "GPU Resources allows Marathon to get offers from Mesos agents with GPUs. For details, see MESOS-5634."